odl/solvers/util/callback.py

```diff
@@ -416,25 +423,37 @@ def __init__(self, fmt='Time elapsed = {:<5.03f} s', step=1):
             where ``runtime`` is the runtime since the last iterate.
         step : positive int, optional
             Number of iterations between prints.
+
+        Other Parameters
+        ----------------
+        kwargs :
+            Key word arguments passed to the print function.
         """
         self.fmt = str(fmt)
         self.step = int(step)
         self.iter = 0
-
-        self.time = time.time()
+        self.cumulative = cumulative
+        self.base_time = time.time()
+        self.time = 0
+        self.kwargs = kwargs
 
     def __call__(self, _):
         """Print time elapsed from the previous iteration."""
         if self.iter % self.step == 0:
             t = time.time()
-            print(self.fmt.format(t - self.time))
-            self.time = t
+            self.time = t - self.base_time
```

Comment:
Is there any reason why we need to save this as `self...`?

Reply:
I was thinking that it was good to have the cumulative time as an output attribute but actually it is cleaner for this not to be. This is now fixed.